### DISCUSSION TITLE
intellij import

### DISCUSSION THREAD

**Discussion by cagneymoreau** [jzy3d/jzy3d-api#145] [General]
This might seem dumb but I spent at least 30 minutes trying to import this project on intellij.
 The correct cycle of events is to
 open an existing unrelated project ->
file->new project from version control -> 
close the imported project window
 from the unrelated project ->file-> new project from existing sources -> select the new projects location
from new project select top-> file -> add framework-> maven
file-> set module language level

**Comment by jzy3d** [comment]
Sorry for late reading, I have not been notified by your post. 
Thanks for sharing this, I should add this to the readme.
